### ISSUE TITLE
the provided pretrained weight only includes the encoder weight, not includes the decoder weight

### ISSUE THREAD

**Issue by wangqingbin** [facebookresearch/mae#8]
It seems that the provided pretrained weight only includes the encoder weight, not includes the decoder weight, could you provide the entire weight for pretraining model. Thank you very much!

**Comment by KaimingHe** [comment]
See https://dl.fbaipublicfiles.com/mae/pretrain/mae_pretrain_vit_base_full.pth. The other two files mae_pretrain_vit_large_full.pth and mae_pretrain_vit_huge_full.pth are in the same dir too. Note these models reconstruct normalized pixels.

**Comment by tristanengst** [comment]
Could you confirm the hyperparameters used for training the model linked above? I need to train them for a few more epochs so as to compare with a related method I'm working on. Thanks so much!
